doc/api/v8.md

```diff
@@ -87,6 +87,24 @@ The value returned is an array of objects containing the following properties:
 ]
 ```
 
+## v8.getHeapdump()
```

Comment:
Bikeshedding: why is it not takeHeapSnapshot()? That’s closer to the file extension and the v8 API. The word heapdump is not new of course but this is introducing new terminology for no particular good reason?

Also is it necessary to provide the write method separately? If it’s necesary to have a way to write the file to disk synchronously, can’t that be a method of the returned object (that result in two JS to C++ boundary crossing but I don’t suppose it’s a big deal?) instead of a separate method hanging off v8?

(Sorry for all these questions but unless this is made experimental the API design is going to stick and it could be hard to extend in the future if people start to depend on them in production)

Reply:
> Also is it necessary to provide the write method separately?

I think so yes. In a case where what someone wants is to grab a snapshot on `process.on('exit')`, having the separate `writeHeapdump()` option that does not allocate and return a new object just to get the file serialized out is not ideal. I would bet that the typical case for the majority of users will be to just call `writeHeapdump()` and returning the object is wasteful.